api/primary_test.go

```diff
@@ -56,10 +57,45 @@ func TestCorsRequest(t *testing.T) {
 		t.Log(k, " : ", v)
 	}
 
+	// test response status code
 	if w.Code == 404 {
```

Comment:
I think I originally wrote this bit and I don't remember what I was thinking at the time. Do we know what the response code should be? Rather than just 'not 404', I'd like it to be changed to the exact code it is, but if we're not sure, I don't mind.